README.md

```diff
@@ -0,0 +1,180 @@
+<!--
+SPDX-FileCopyrightText: 2020 Serokell <https://serokell.io/>
+
+SPDX-License-Identifier: MPL-2.0
+-->
+
+# deploy-rs
+
+A Simple, multi-profile Nix-flake deploy tool.
+
+**This is very early development software, you should expect to find issues, and things will change**
+
+## Usage
+
+Basic usage: `deploy [options] <flake>`.
+
+The given flake can be just a source `my-flake`, or optionally specify the node to deploy `my-flake#my-node`, or specify a profile too `my-flake#my-node.my-profile`.
+
+You can try out this tool easily with `nix run`:
+- `nix run github:serokell/deploy-rs your-flake`
+
+If you require a signing key to push closures to your server, specify the path to it in the `LOCAL_KEY` environment variable.
+
+Check out `deploy --help` for CLI flags! Remember to check there before making one-time changes to things like hostnames.
+
+There is also an `activate` binary though this should be ignored, it is only used internally and for testing/hacking purposes.
+
+## API
+
+### Profile
+
+This is the core of how `deploy-rs` was designed, any number of these can run on a node, as any user (see further down for specifying user information). If you want to mimick the behaviour of traditional tools like NixOps or Morph, try just defining one `profile` called `system`, as root, containing a nixosSystem, and you can even similarly use [home-manager](https://github.com/nix-community/home-manager) on any non-privileged user.
+
+```nix
+{
+  # A derivation containing your required software, and a script to activate it in `${path}/activate`
+  # For ease of use, `deploy-rs` provides a function to easy all this required activation script to any derivation
```

Comment:
Grammar has left the chat

Reply:
I do not remember writing this and I'm checking my carbon monoxide detectors tbh